packages/core/src/utils/createTheme.js

```diff
@@ -151,11 +151,14 @@ const createPalette = ({ palette = {}, ...theme }) => {
 const createTheme = (theme = {}, customBreakpoints = null) => {
   const mergedTheme = deepmerge(defaultTheme, theme)
 
+  const mediaQueries = customBreakpoints
+    ? createMediaQueries(customBreakpoints)
+    : mergedTheme.mediaQueries
   return {
     ...mergedTheme,
     contrastRatio: mergedTheme.contrastRatio || 2.6,
     breakpoints: addAliases(customBreakpoints || mergedTheme.breakpoints),
-    mediaQueries: addAliases(mergedTheme.mediaQueries),
```

Comment:
With this previous logic, only the breakpoints were updated. This means that any DS component would adhere to those new breakpoints. However, the media queries we still the same as the default theme, so they new breakpoints do not work when accessing the diff media query inside a styled component.